docs/src/userguide/language_basics.rst

```diff
@@ -135,9 +189,21 @@ whereas ``x[0]`` is.
 
 Also, the Python types ``list``, ``dict``, ``tuple``, etc. may be used for
 static typing, as well as any user defined :ref:`extension-types`.
-For example::
+For example
+
+.. tabs::
+
+    .. group-tab:: Pure Python
 
-    cdef list foo = []
+        .. code-block:: python
+
+            foo: list = []
```

Comment:
This should better be wrapped in a function to make it actually do something. (Same below.)

Reply:
done.